port/action/schema.go

```diff
@@ -461,7 +460,7 @@ func StringPropertySchema() schema.Attribute {
 			Optional:            true,
 		},
 		"format": schema.StringAttribute{
-			MarkdownDescription: "The format of the string property",
+			MarkdownDescription: "The format of the string property. Common accepted values include `date-time`, `url`, `email`, `ipv4`, `ipv6`, `yaml`, `entity`, `user`, `team`, `proto`, `markdown`",
```

Comment:
```suggestion
			MarkdownDescription: "The format of the string property. Accepted values include `date-time`, `url`, `email`, `ipv4`, `ipv6`, `yaml`, `entity`, `user`, `team`, `proto`, `markdown`",
```
since its not really json-schema complient than its not really "common"